tmt/utils/__init__.py

```diff
@@ -4358,6 +4358,30 @@ def increment(
 
                 raise GeneralError(message) from error
 
+        # Handle GitHub-specific responses
+        # https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#exceeding-the-rate-limit
+        response = cast(requests.Response, kwargs.get('response'))
```

Comment:
Must be `Optional`, `get()` may return `None`.

Reply:
This one is still present.

Reply:
aaand done